pkg/remote/google/util.go

```diff
@@ -0,0 +1,9 @@
+package google
+
+import (
+	"strings"
+)
+
+func trimResourceName(name string) string {
+	return strings.TrimPrefix(name, "//compute.googleapis.com/")
```

Comment:
This should for for every kind of service and not only compute if you name the func like this.

We should either change the behavior to be able to trim any previx regardless of the service type or rename the func name for something like `trimComputeResourceName`

Reply:
With a regex we could match `\/\/[\w]+.googleapis.com\/` WDYT ?

Reply:
> With a regex we could match \/\/[\w]+.googleapis.com\/ WDYT ?

We should be sure that any service ends with `.googleapis.com` 

Another solution could be to remove every prefix `^\/\/.*?\/`, I don't know if it can be dangerous ...

Reply:
Yes this is a common suffix for all resource types (except for k8s). See https://cloud.google.com/asset-inventory/docs/resource-name-format

We can also create a function for each service and change that later when we'll have more experience with GCP

Reply:
Ok so I think `^\/\/[\w]+.googleapis.com\/` can do the job (do not forget the `^` to match the beginning of the string) since everything ends with `googleapis.com`.

I think you missread since kube related resources are under `container.googleapis.com`

![image](https://user-images.githubusercontent.com/6154987/134022609-bb9435d8-ed94-4fbc-ba43-3334b93cf3a1.png)

Reply:
> I think you missread since kube related resources are under container.googleapis.com

yes indeed